### ISSUE TITLE
The CTA card in Notification Settings has the wrong size

### ISSUE THREAD

**Issue by taipeicoder** [Automattic/wp-calypso#94049]
See pbxlJb-6dR-p2, 6:40.

@ianstewart pointed out that this card in the Notification Settings page has the wrong size. @lucasmendes-design could you take a look at this?

![Image](https://github.com/user-attachments/assets/26c524a9-a13b-4f62-9afa-a452bc9372da)

**Comment by javierarce** [comment]
I don't think that card has the wrong font size: the text is `16px`, which is the same in other setting pages, but we could tweak the design a bit to make it look more consistent.

![image](https://github.com/user-attachments/assets/0ff1f4eb-52af-493f-8d0a-f8501d363831)
![image](https://github.com/user-attachments/assets/e87db205-78ac-47aa-a83c-cf0e71a8ab19)

These are the changes I made:

1. I added section headers to the top of the two boxes.
2. I replaced the button with a toggle.
3. I improved the warning message to fit one line, the case of the link, and used the warning color.

If we modify this, we should apply the first change to the rest of the Notification settings.

Figma file: wCm5Tzb7JBaG4WNowWbjES-fi-5266_56091
